public/app/features/trails/DataTrail.test.tsx

```diff
@@ -22,13 +21,13 @@ describe('DataTrail', () => {
     );
   });
 
-  describe('Given starting trail with url sync and no url state', () => {
+  describe('Given starting non-embedded trail with url sync and no url state', () => {
     let trail: DataTrail;
+    const preTrailUrl = '/';
 
     beforeEach(() => {
       trail = new DataTrail({});
-      locationService.push('/');
-      getUrlSyncManager().initSync(trail);
```

Comment:
Since the external DataTrailsApp isn't doing this for the lifetime of the trail, we need to remove this from the test to ensure that we are syncing the urls with state at the key points.